### PR TITLE
Add 0.82 release blog

### PR DESCRIPTION
**Pull request by ericgregory**
Adding release blog for 0.82.

### CODE REVIEW

File: blog/wasmcloud-0.82-wasi-p2-is-here/082.png
Comment (lachieh) [inline blog/wasmcloud-0.82-wasi-p2-is-here/082.png:1]: love it. 1200x630 for blog images please 🙏🏻

File: blog/wasmcloud-0.82-wasi-p2-is-here/index.mdx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+WebAssembly components&mdash;portable, interoperable, and composable&mdash;are the future of Wasm artifacts, and with WASI 0.2, that future is here. Components are ideally suited to wasmCloud's design philosophy, and we're all-in: as of 0.82, the old “module + wasmbus” model is deprecated and will be removed in 1.0.  
+
+Version 0.82 supports running both components and legacy modules, so now is the time to migrate old projects, do it with stable WASI 0.2 APIs, and have some fun!
+
+# Transitioning to components
+
+It's easier than ever to start a new project in the language of your choice. We've added project templates for building WebAssembly components from Python and TypeScript, which join Rust and TinyGo as first-class citizens in the `wash` CLI. When you run `wash new actor` you can choose from the following options:
+
+```shell
```
Comment (lachieh) [inline blog/wasmcloud-0.82-wasi-p2-is-here/index.mdx]:
```suggestion
```bash
```

File: blog/wasmcloud-0.82-wasi-p2-is-here/index.mdx [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+      // highlight-end
+```
+
+# OpenTelemetry logging
+
+OpenTelemetry (OTEL) logging joins our existing tracing support to provide more options for observability. (Metrics will complete the trifecta in wasmCloud 1.0.)  
+
+By default, plaintext logs are written to `stderr` as before. You can enable structured logging and emit JSON as an OTEL signal by launching a wasmCloud host with the OTEL exporter enabled via environment variable:
+
+```shell
```
Comment (lachieh) [inline blog/wasmcloud-0.82-wasi-p2-is-here/index.mdx]:
```suggestion
```bash
```

File: blog/wasmcloud-0.82-wasi-p2-is-here/index.mdx [hunk 3]
```diff
@@ -0,0 +1,85 @@
+---
+title: "wasmCloud 0.82: WASI 0.2, OTEL logging, and more"
+date: 2024-02-20T10:00:00-07:00
```
Comment (brooksmtownsend) [inline blog/wasmcloud-0.82-wasi-p2-is-here/index.mdx]: nit: change to today 😄 
```suggestion
date: 2024-02-21T10:00:00-07:00
```

**Review by lachieh — APPROVED** [review]
Love a good em dash. Only some small functional nits.

File: blog/wasmcloud-0.82-wasi-p2-is-here/index.mdx [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+Previously, you'd have to know or find the absolute path to your signed build artifact&mdash;an extra bit of friction that just didn't need to be there. Now you can stay in the context of your project and point to the relative path, and the spec for an actor can look like this:
+
+```yaml
+- name: hello
+      type: actor
+      properties:
+      // highlight-start
+        image: file://./build/hello_s.wasm
+      // highlight-end
```
Comment (brooksmtownsend) [inline blog/wasmcloud-0.82-wasi-p2-is-here/index.mdx]: This indentation is a little weird, suggesting what may fix it
```suggestion
- name: hello
  type: actor
  properties:
      // highlight-start
    image: file://./build/hello_s.wasm
      // highlight-end
```

**Review by brooksmtownsend — CHANGES_REQUESTED** [review]

**Review by brooksmtownsend — APPROVED** [review]

**Review by lachieh — APPROVED** [review]

**Pull request merged**